openfecwebapp/models/candidates.py

```diff
@@ -0,0 +1,24 @@
+from openfecwebapp.models.shared import committee_type_map
+from marshmallow import Schema, fields, pprint, pre_load
+
+
+def handle_errors(schema, errors, obj):
+    raise ValueError('An error occurred:\n{} \n\n...while marshalling the data:\n{}\n\n...for schema: {}'.format(errors, obj, schema))
+
+
+class CandidateSchema(Schema):
+    """ Defines a schema for candidate.
+    Only fields which need special treatment (renaming, etc.)
+    need be defined separately. All others may go in Meta.fields.
+    """
+    incumbent_challenge = fields.Str(allow_none=True)
```

Comment:
The `allow_none` option only has an effect for deserializing, which I don't think we're doing. Can these fields move into `Meta#fields` below?